### PR TITLE
porting rqt_console to ros2

### PR DESCRIPTION
**Pull request by mlautman**
This ports rqt_console to ros2

This port uses the log msg defined here: https://github.com/ros2/rcl_interfaces/blob/dc61c47dca17e2ee5f8f30993054c5c75a050ea3/rcl_interfaces/msg/Log.msg

Log msg pr: https://github.com/ros2/rcl_interfaces/pull/53
issue: https://github.com/ros2/rcl_interfaces/issues/50

It also depends on: https://github.com/ros-visualization/rqt/pull/161

### CODE REVIEW

File: src/rqt_console/console.py [hunk 1]
```diff
@@ -82,6 +83,8 @@ def __init__(self, context):
 
         self._subscriber = None
         self._topic = '/rosout_agg'
+        self._node = rclpy.create_node('rqt_console')
```
Comment (brawner) [inline src/rqt_console/console.py]: This should make use of the new ros2_plugin_context._node

File: src/rqt_console/console_settings_dialog.py [hunk 2]
```diff
@@ -32,11 +32,13 @@
...
 import os
 
+from ament_index_python.resources import get_resource
+
 from python_qt_binding import loadUi
 from python_qt_binding.QtWidgets import QDialog
 
-from rqt_logger_level.logger_level_widget import LoggerLevelWidget
-from rqt_logger_level.logger_level_service_caller import LoggerLevelServiceCaller
+# from rqt_logger_level.logger_level_widget import LoggerLevelWidget
```
Comment (brawner) [inline src/rqt_console/console_settings_dialog.py:41]: Mark as todo if this needs to be re-implemented

Comment (mlautman) [reply src/rqt_console/console_settings_dialog.py:41]: done

File: src/rqt_console/console.py (hunk 1 above)
Comment (mlautman) [reply src/rqt_console/console.py]: done

File: CMakeLists.txt [hunk 3]
```diff
@@ -1,18 +1,25 @@
...
-# Load catkin and all dependencies required for this package
-find_package(catkin REQUIRED)
-catkin_package()
-catkin_python_setup()
+
+# Load ament and all dependencies required for this package
+find_package(ament_cmake REQUIRED)
+find_package(ament_cmake_python REQUIRED)
+# Note: This is needed for rcl_interfaces Log.msg
+find_package(rcl_interfaces REQUIRED)
```
Comment (dirk-thomas) [inline CMakeLists.txt:8]: See #9.

**Pull request merged**